src/flask_track_usage/__init__.py

```diff
@@ -149,9 +151,23 @@ def after_request(self, response):
             'date': int(time.mktime(now.timetuple()))
         }
         if self._use_freegeoip:
-            ip_info = json.loads(urllib.urlopen(
-                self._freegeoip_endpoint + urllib.quote_plus(
-                    remote_addr)).read())
+            clean_ip = urllib.quote_plus(str(ctx.request.remote_addr))
```

Comment:
This will fail on Python 3 though I realize the earlier version would have as well. I recommend using the following import to support both versions:

```python
from six.moves.urllib_parse import quote_plus
```